piezo_web_app/PiezoWebApp/src/handlers/get_jobs.py

```diff
@@ -0,0 +1,21 @@
+from tornado_json import schema
+
+from PiezoWebApp.src.handlers.base_handler import BaseHandler
+
+
+# pylint: disable=abstract-method
+class GetJobsHandler(BaseHandler):
+    @schema.validate(
+        output_schema={
+            "type": "object",
+            "properties": {
+                "message": {"type": "string"},
+            },
+        }
```

Comment:
This doesn't match up with what should be returned (missing `"spark_jobs"`, I think)

Reply:
2918bdd